assets/crx/html/embed.html

```diff
@@ -15,6 +15,14 @@
         <input type="text" id="find-text" placeholder="Find in page" />
     </div>
 
+    <!-- Alert/Confirm box -->
+    <div id="dialog-box">
+      <div>Example dialog text</div>
```

Comment:
Please add `id=` or `class=` to the `div`, `input`, and two `button` elements, as referring to them via their index or tag name is risky and may break in the future if we introduce other elements here.